Sources/AsyncHTTPClient/HTTPHandler.swift

```diff
@@ -664,12 +664,12 @@ internal struct TaskCancelEvent {}
 
 internal class TaskHandler<Delegate: HTTPClientResponseDelegate>: RemovableChannelHandler {
     enum State {
-        case idle
-        case bodySent
-        case sent
-        case head
+        case inactive
+        case sending_waiting
```

Comment:
- we should use `lowerCamelCase` to match swift's usual style.
- I think we can do better than `sendingWaiting`, ie. what are we sending and what are we waiting for?

Reply:
done